dsc/cli.py

```diff
@@ -27,20 +28,33 @@
     "workflow run files",
     required=True,
 )
+@click.option(
+    "-r",
+    "--create-and-send-report",
+    is_flag=True,
+    help=(
+        "Pass to create and send summary report to recipients. "
+        "Reports generated for 'reconcile' and 'submit' steps must only be sent to "
+        "DSC admins."
+    ),
+    default=False,
+)
 @click.option(
     "-v", "--verbose", is_flag=True, help="Pass to log at debug level instead of info"
 )
 def main(
     ctx: click.Context,
     workflow_name: str,
     batch_id: str,
+    create_and_send_report: bool,  # noqa: FBT001
```

Comment:
Is there a reason to put this on `main` rather than the CLI command itself? Then there would be no need to add it to the context object

Reply:
See https://github.com/MITLibraries/dspace-submission-composer/pull/121#discussion_r1946871007.